python_modules/dagster/dagster/core/system_config/composite_descent.py

```diff
@@ -176,8 +176,8 @@ def _get_mapped_solids_dict(
     with user_code_error_boundary(
         DagsterConfigMappingFunctionError, _get_error_lambda(current_stack)
     ):
-        mapped_solids_config = graph_def.config_mapping.config_fn(
-            config_mapped_solid_config.value.get("config", {})
+        mapped_solids_config = graph_def.config_mapping.resolve(
+            config_mapped_solid_config.value.get("config", {}), config_has_been_validated=True
```

Comment:
how do we handle enums in composite descent? is it under test at all?

Reply:
If you don't  want to chase this down in this PR, just add a comment and keep calling config_fn directly